toolkit/tools/pkggen/worker/create_worker_chroot.sh

```diff
@@ -7,18 +7,26 @@
 # $3 path to find RPMs. May be in PATH/<arch>/*.rpm
 # $4 path to log directory
 
-[ -n "$1" ] && [ -n "$2" ] && [ -n "$3" ] && [ -n "$4" ] || { echo "Usage: create_worker.sh <./worker_base_folder> <rpms_to_install.txt> <./path_to_rpms> <./log_dir>"; exit; }
+[ -n "$1" ] && [ -n "$2" ] && [ -n "$3" ] && [ -n "$4" ] && [ -n "$5" ] && [ -n "$6" ] || { echo "Usage: create_worker.sh <./worker_base_folder> <rpms_to_install.txt> <./path_to_rpms> <./log_dir> <./bldtracker>"; exit; }
 
 chroot_base=$1
 packages=$2
 rpm_path=$3
 log_path=$4
+bldtracker=$5
+timestamp_dir=$6
 
 chroot_name="worker_chroot"
 chroot_builder_folder=$chroot_base/$chroot_name
 chroot_archive=$chroot_base/$chroot_name.tar.gz
 chroot_log="$log_path"/$chroot_name.log
 
+$bldtracker \
+    --script-name="create_worker_chroot.sh" \
+    --step-name="test step" \
+    --file-path=$timestamp_dir \
+    --mode="n"
+
 install_one_toolchain_rpm () {
```

Comment:
Ideally, we should track each package being installed inside this loop, this is the bulk of the time spent in this tool.

Reply:
Adding this feature will change the number of timestamp entries for create_worker_chroot.csv, which impacts some functionality in dashboard.go. We will address this issue in the next PR.